x-pack/plugins/security_solution/public/entity_analytics/common/utils.ts

```diff
@@ -26,6 +27,27 @@ export const RISK_SEVERITY_COLOUR: { [k in RiskSeverity]: string } = {
   [RiskSeverity.Critical]: SEVERITY_COLOR.critical,
 };
 
+export const useRiskSeverityColors = (): { [k in RiskSeverity]: string } => {
```

Comment:
ℹ️ FYI, there were conversations about severity colors and what they should be mapped to (([here](https://github.com/elastic/kibana/pull/204352#discussion_r1886896464))).

We're adding new tokens for this ([PR](https://github.com/elastic/eui/pull/8247)), but it's not blocking for proceeding here, in case we're updating as follow-up.